### PR TITLE
File card helper

### PR DESCRIPTION
**Pull request by lgavillet**

### CODE REVIEW

File: lib/mtl/rails/card_file_presenter.rb [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+    #                                    'data-mtl-delete': '/path/to/delete/the/file')
+    # ```
+    # ```html
+    # <a class="card-panel" href="/path/to/file.jpg" target="_blank" title="Document Dolorem.jpg">
+    #   Document Dolorem.jpg
+    #   <span class="grey-text">
+    #     <i class="material-icons red-text">image</i>
+    #     JPG
+    #   </span>
+    #   <a href="/path/to/delete/the/file" method="delete">
```
Comment (lwe) [inline lib/mtl/rails/card_file_presenter.rb]: we can't have an `<a>` within an `<a>`, so we need to handle the delete via JS, sadly.

File: lib/mtl/rails/card_file_presenter.rb [hunk 2]
```diff
@@ -0,0 +1,132 @@
+require 'uri'
+
+module Mtl
+  module Rails
+    # Specialised renderer for the file card element used in a file collection
+    #
+    # Basic usage:
```
Comment (lwe) [inline lib/mtl/rails/card_file_presenter.rb]: I guess the "public" API is `mtl_card_file`, right? So I propose to move this documentation to the `helpers.rb` and mark this one as private with a short description (one sentence) or what it's used for.

File: lib/mtl/rails/card_file_presenter.rb (hunk 1 above)
Comment (lgavillet) [inline lib/mtl/rails/card_file_presenter.rb]: AAAaaah true, what a mongo, yep....

**Comment by lwe** [comment]
Thanks.

**Pull request merged**